doc/news/DM-41111.feature.rst

```diff
@@ -0,0 +1 @@
+creating mtcs methods to run the M2 bump test
```

Comment:
Please, add punctuation and capitalize the first letter in "creating". However, I'd rephrase it slightly though. Maybe something like,

```
Update ``MTCS`` class to support running M2 bump tests.
```